fdbserver/VersionedBTree.actor.cpp

```diff
@@ -6298,9 +6524,9 @@ class VersionedBTree {
 		PathEntry& back() { return path.back(); }
 		void popPath() { path.pop_back(); }
 
-		Future<Void> pushPage(const BTreePage::BinaryTree::Cursor& link) {
+		Future<Void> pushPage(PagerEventReasons r, unsigned int l, const BTreePage::BinaryTree::Cursor& link) {
```

Comment:
This `pushPage()` is always used for non-root nodes because requires a `link` from an existing BTree node, so it doesn't need to accept a `level` parameter.  The level of the page being pushed onto the path is `path.back().btPage()->height - 1`